server/server.go

```diff
@@ -307,6 +307,7 @@ func (r *rodeServer) ListResources(ctx context.Context, request *pb.ListResource
 		r.esClient.Search.WithIndex(rodeElasticsearchOccurrencesAlias),
 		r.esClient.Search.WithBody(encodedBody),
 		r.esClient.Search.WithSize(maxPageSize),
+		r.esClient.Search.WithSort("resource.uri:asc"),
```

Comment:
Could/should this use the [sort](https://github.com/rode/grafeas-elasticsearch/blob/fbd2243fbf82e6c40ff5ff35297b75a3fcfa3194/go/v1beta1/storage/esutil/types.go#L51) field on the `EsSearch` struct now?

Reply:
It sure can!